test-app/src/app/global-error-handler.ts

```diff
@@ -0,0 +1,8 @@
+import { ErrorHandler, Injectable } from "@angular/core";
+
+@Injectable()
+export class GlobalErrorHandler implements ErrorHandler {
+  handleError(error: Error) {
+    console.log(error.toString());
```

Comment:
Is this on purpose console.log and not console error?

Reply:
Will check if the stderr stream is bridged to the Xcode console log and get back to you.